packages/azure-http-specs/specs/azure/resource-manager/resources/mockapi.ts

```diff
@@ -60,6 +61,134 @@ const validSingletonResource = {
   },
 };
 
+const validLocationResource = {
+  id: `/subscriptions/${SUBSCRIPTION_ID_EXPECTED}/providers/Azure.ResourceManager.Resources/locations/${LOCATION_EXPECTED}/locationResources/resource`,
```

Comment:
It is not consistent on `/subscriptions/##`.

This API is `/subscriptions/${SUBSCRIPTION_ID_EXPECTED}`, while others are `/subscriptions/:subscriptionId` + params

Make it consistent with other tests (subscriptionId should be in all ARM tests, check these)

Reply:
Similar for the `/locations/${LOCATION_EXPECTED}`

Reply:
To me, having e.g. `/subscriptions/:subscriptionId/providers/Azure.ResourceManager.Resources/locations/:location/locationResources/:locationResourceName` and then param be
```
    params: {
      subscriptionId: SUBSCRIPTION_ID_EXPECTED,
      location: LOCATION_EXPECTED,
```
appears easier to understand.

Reply:
The `validLocationResource` is an expected value constant, `/subscriptions/:subscriptionId` should not appear in this constant.